Add brief details about and link to, windbg's .kdfiles command (#1493) (#1578)
microsoft/ebpf-for-windows: docs/GettingStarted.md

Otherwise, after installing the nuget package, as a one-time operation, you will
currently need to run the `export_program_info.exe` tool to complete the install. This
tool can be found in your project's `packages\eBPF-for-Windows\build\native\bin` directory.

> If you are working on the EbpfCore or the NetEbpfExt drivers, you may find the WinDbg command ```.kdfiles``` to be useful. This

Reviewer: Does this also apply when using the VS [Debugger Immediate Window](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/entering-debugger-commands-in-visual-studio)?  If not, then update the first part of the sentence, as in:

```suggestion
If you are using WinDbg to work on the EbpfCore or the NetEbpfExt drivers, you may find the WinDbg command ```.kdfiles``` to be useful. This
```

If kdfiles does in fact work in the VS debugger immediate window, then update latter half of the sentence to not be limited to WinDbg.

Author: Driver debugging using VS requires the installation of the WDK on the test device [(for VS intergration).](https://learn.microsoft.com/en-us/windows-hardware/drivers/gettingstarted/provision-a-target-computer-wdk-8-1).  Unfortunately, 'Note' @ the [WDK install page](https://learn.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk) mentions that driver debugging under VS is currently non-functional and instead recommends using WinDbg instead.

I've updated this verbiage accordingly.

command allows the replacement of a driver binary on the target machine with another binary from the machine WinDbg is running
on (typically the development machine) at driver load time.

This eliminates the need for repeated manual copy of the modified driver binary and saves considerable time during the

> development cycle.  More details are [here](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/-kdfiles--set-driver-replacement-map-)

Reviewer: ```suggestion
development cycle.  See the [Windows Hardware Developer documentation](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/-kdfiles--set-driver-replacement-map-) for more details.
```

Explanation: never use "here" as link text, as it violates [W3C guidelines](https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text) which are basically for accessibility (screen readers, etc.).

Author: Huh. Had never thought of that.  Thanks for pointing this out! Fixed.
